### DISCUSSION TITLE
Welcome to tinier11 Discussions!

### DISCUSSION THREAD

**Discussion by bravomail** [bravomail/tinier11#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nullacron** [comment]
What is this compact flag?

**Reply by bravomail** [reply]
compact.exe has /compactOS flag

**Comment by Reynolds-Beta** [comment]
All I can say is clever. I've been doing much of this by hand over time.
